### PR TITLE
Improve docs of demandlib

### PR DESCRIPTION
**Pull request by uvchik**
* [x] description of electricity demand profiles
 * ~[ ] usage~

Docs can be found [here.](https://demandlib.readthedocs.io/en/docs-improve-docs/description.html#electrical-profiles)

### CODE REVIEW

**Comment by uvchik** [comment]
I will not add the usage, because the usage chapter will be created in PR #45. We should merge this first, if its fine for you. I changed the structure of the docs a bit but mainly added content. Nothing fundamental. Proofing would be good, though.

File: docs/bdew.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+Depending on the profile type, different coefficients A, B, C, D for the sigmoid function are used.
+
+.. math::
+
+   h(\theta) &= \frac{A}{1+(\frac{B}{\theta-\theta_0})^C} + D \\
+
+   \theta_0 &= 40^\circ C
+
+Types of houses:
```
Comment (p-snft) [inline docs/bdew.rst:44]: As far as I understand, the profiles include space heating (SH) and domestic hot water (DHW). Thus, they become inaccurate if the insulation is different from the one originally assumed: Demand in summer is dominated by DHW, demand in summer may or may not be dominated by SH - depending on the insulation. (This is also the way, I interpret https://github.com/oemof/demandlib/issues/20.)

Maybe, this limitation should be made explicit.

**Review by p-snft — APPROVED** [review]
The verbosity is very adequate and I don't see any mistakes. Thanks. (Still, you might consider adding a sentence about the limitations of the profiles.)

File: docs/bdew.rst (hunk 1 above)
Comment (uvchik) [reply docs/bdew.rst:44]: I just revised the general structure and added some information about electrical load profiles. I did not touch the content of the heat profiles.

**Comment by uvchik** [comment]
I added an issue for further ToDo's #48.

**Pull request merged**